gradle/libs.versions.toml

```diff
@@ -12,7 +12,7 @@ gradlePlugin = "8.1.0"
 lintMinCompose = "30.0.0"
 
 ktlint = "0.45.2"
-kotlin = "1.9.0"
```

Comment:
Not blocking - is bumping Kotlin and the compiler require from accompanist? Just wondering if the transitive Kotlin bump here is worth it (or if that bump would be transitive anyway from base Compose)

Reply:
We always just release with the latest compiler to make sure we stay up to date with fixes there. I haven't heard an issue yet with an incompatible change